### ISSUE TITLE
Catalog builder in a Docker container

### ISSUE THREAD

**Issue by aradhakrishnanGFDL** [aradhakrishnanGFDL/CatalogBuilder#46]
An automated workflow to build a docker image of the catalog builder.
Ease of use for users using any platform.

**Comment by aradhakrishnanGFDL** [comment]
@Ciheim-- 

This needs testing, and notes documented before the issue can be closed.
Once manual testing is complete, close this issue. Open new issue to update GitHub actions workflows to include an automated test for the docker container.

Manual testing

Run:

docker pull aparnadotnoaa/catalogbuilder-image:latest

make sure it's public
make sure it can generate a GFDL catalog (meaning, it can load the env and run the gfdl wrapper script successfully, generating json and csv) 
Sanity testing needed for json and csv, just diff with the same experiment for which catalog was generated using the conda package, or the git repo.

Update testing notes in this issue with the docker run or exec commands.
Update README for users to use this container at GFDL, and outside of GFDL.

**Comment by aradhakrishnanGFDL** [comment]
awaits testing
